containers/daap-create-metadata/src/var/task/create_metadata.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+
+from data_platform_logging import DataPlatformLogger
+from data_product_metadata import DataProductMetadata
+
+
+def handler(event, context):
+    logger = DataPlatformLogger(
+        extra={
+            "image_version": os.getenv("VERSION", "unknown"),
```

Comment:
if we are only using the logger in lambdas, the image version and base image version can be set in the logger object and we don't have to have this code at the start of every lambda

Reply:
yeah i think that could work, part of the wider logger refactor? You mentioned changing the way it writes the json too which i think would be good. See you've created the issue for it already

Reply:
yea, maybe I should pick it up sooner rather than later